.github/workflows/build.yaml

```diff
@@ -102,19 +102,19 @@ jobs:
         - uses: actions/checkout@v3
           with:
             submodules: recursive
-        - name: Download leap-dev.deb (Ubuntu 20 only)
-          if: matrix.platform == 'ubuntu20'
+        - name: Download leap-dev.deb (Ubuntu 22 only)
+          if: matrix.platform == 'ubuntu22'
           uses: AntelopeIO/asset-artifact-download-action@v3
           with:
             owner: AntelopeIO
             repo: leap
             file: 'leap-dev.*(x86_64|amd64).deb'
```

Comment:
I think we should change this to `'leap-dev.*ubuntu22\.04_amd64.deb'` as has been done other places, for example,
https://github.com/AntelopeIO/reference-contracts/blob/732fa98cfc632c56fab9c8ab81786d51fffa6e3e/.github/workflows/build.yaml#L78
https://github.com/eosnetworkfoundation/eos-evm-contract/blob/5e8c9aac043e27d737075e65f636847bf0a6d7e3/.github/workflows/contract.yml#L72

That will reduce ambiguity in the case where multiple leap-dev.deb packages exist in a single `container-package` (unfortunate that in the current design `container-package` can contain multiple files vs `artifact-name` which is just one file). I realize it's not a problem now but it might be in the future.

Reply:
Makes sense, is it always `leap-dev.*ubuntu22\.04_amd64.deb` or is  `leap-dev.*ubuntu22\.04_(x86_64|amd64).deb` the better match.